cmd/jnet/main.go

```diff
@@ -30,17 +32,17 @@ func main() {
 }
 
 func run() error {
-	if err := os.MkdirAll("tmp", 0700); err != nil {
+	if err := os.MkdirAll("tmp", filePerm); err != nil {
```

Comment:
No merece la pena como constante, aparte de que filePerm hubiera sido un nombre muy malo
```suggestion
	if err := os.MkdirAll("tmp", 0700); err != nil {
```